cajparser.py

```diff
@@ -1,5 +1,6 @@
 import os
 import struct
+import imagesize
```

Comment:
I would suggest doing import only when needed (ie if the width height look obviously wrong).